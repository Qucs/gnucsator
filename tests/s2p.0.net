Pac:P1 _net0 gnd Num="1" Z="50 Ohm" P="0 dBm" f="1 GHz" Temp="26.85"
Pac:P2 _net5 gnd Num="2" Z="50 Ohm" P="0 dBm" f="1 GHz" Temp="26.85"
# Eqn:Eqn2 T="twoport(S,'S','T')" Export="yes"
SPfile:line _net0 _net5 gnd File="line.0.s2p"
.SP:SP1 Type="log" Start="0.5e9" Stop="6e9" Points="11" Noise="no" NoiseIP="1" NoiseOP="2" saveCVs="no" saveAll="no"
