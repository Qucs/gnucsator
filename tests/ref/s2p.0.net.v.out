
module main ();
Pac #(.Num(1),.P(0 ),.Temp(26.85),.Z(50),.f(1G)) P1 (.1(_net0),.2(0));
Pac #(.Num(2),.P(0 ),.Temp(26.85),.Z(50),.f(1G)) P2 (.1(_net5),.2(0));
SPembed #(.File(File)) line (.p0(_net0),.p1(_net5),.p2(0));
endmodule // main

